python/ray/dataframe/test/test_dataframe.py

```diff
@@ -954,10 +954,31 @@ def test_as_blocks():
 
 
 def test_as_matrix():
-    ray_df = create_test_dataframe()
+    test_data = TestData()
```

Comment:
Can you use the fixture model for testing here and define the numpy matrix in the tests to compare against?

Reply:
Why does this need a fixture model?

Reply:
fixture models are more in tune with what we have been using. The simplest way to do this test would be to run `to_matrix` or `__array__` on both `pd_df` and `ray_df` and check equality.

Reply:
We will make a pass over the tests to unify them in a later PR.

Reply:
Sounds good.